### ISSUE TITLE
Undocumented fileName parameter and -out requirement for plan command

### ISSUE THREAD

**Issue by mloskot** [microsoft/azure-pipelines-terraform#195]
In https://github.com/microsoft/azure-pipelines-terraform/issues/117#issuecomment-1342795831 @mericstam explains that

> for V3 you need to do plan with `commandoptions: -out`

and 

presents example with `outputTo` and `fileName` use for `command: show`.

These are not documented in the [TerraformTaskV3/README.md](https://github.com/microsoft/azure-pipelines-terraform/blob/94890a9b4bddf9df4de6949794956aca1f66113a/Tasks/TerraformTask/TerraformTaskV3/README.md).

A reader may expect to be able to use `outputTo` and `fileName` for `command: plan` too, and not sticking `-out` to `commandOptions`
In other words, a reader may expect unified interface of all TerraformTaskV3 commands, and rightly so, but apparently that is not the case.
IMHO, documentation is unclear there.

**Comment by mericstam** [comment]
Hi, 
I agree docs are a bit unclear. I have planned to redo the docs. trying to fit it into my schedule
